api/tests/devcomp/unit/scripts/get-modules-csv_test.js

```diff
@@ -0,0 +1,147 @@
+import { _getTotalElementsCount } from '../../../../scripts/modulix/get-modules-csv.js';
+import { catchErrSync, expect } from '../../../test-helper.js';
+
+describe('Unit | Scripts | Get Modules as CSV', function () {
+  describe('#getTotalElements', function () {
```

Comment:
```suggestion
  describe('#_getTotalElementsCount', function () {
```

Par ailleurs je trouve toujours bizarre de tester unitairement une méthode privée… mais je n'ai pas mieux à proposer.

Reply:
On laisse comme ça pour l'instant, comme vu ensemble.